modules/ROOT/pages/salesforce/salesforce-connector-studio.adoc

```diff
@@ -0,0 +1,165 @@
+** <<OAuth Username Password>>
++
+. For each of connection choices, you can set optional proxy values:
++
+** *Host* - Host name of the proxy server.
+** *Port* - The port number the proxy server runs on.
+** *Username* - The username to log in to the server.
+** *Password* - The corresponding password.
+
+After setting the *Global Element Properties* screen, return to the main connector menu and set the <<Connector Property Values>>.
+
+== Basic Username Password
+
+Important fields:
+
+* *Username* (required): Enter the Salesforce username.
+* *Password* (required): Enter the corresponding password.
+* *Security token*: Enter the corresponding security token.
+
+== OAuth 2.0
+
+Important fields:
+
+* *Display* (required): How to optimize the display: `PAGE` = Full-page authorization screen (default), `POPUP` = Compact dialog optimized for modern web browser popup windows, and `TOUCH` = Mobile-optimized dialog designed for modern smart phones, such as Android and iPhone.
+* *Consumer key* (required): The consumer key for the Salesforce connected app. See <<Create a Consumer Key>>.
+* *Consumer secret* (required): The consumer secret for the connector to access Salesforce.
+* *Listener config* (required): Configuration for the listener, for example, `HTTP_Listener_config`.
+* *Callback path* (required): Path for the callback, for example, `/callback`.
+* *Authorize path* (required): Path for authorization, for example, `/authorize`.
+* *External callback url*: Callback URL, for example, `+http://localhost:8085/callback+`.
+
+== OAuth JWT
+
+Important fields:
+
+* *Consumer key* (required): The consumer key for the Salesforce connected app. See xref:salesforce-connector-config-topics.adoc#create-consumer-key[Create a Consumer Key].
+* *Key store* (required): See xref:salesforce-connector-config-topics.adoc#keystoregen[Generate a Keystore File].
+* *Store password* (required): The password for the keystore.
+* *Principal* (required): The password for the keystore.
+
+== OAuth SAML
+
+Important fields:
+
+* *Consumer key* (required): The consumer key for the Salesforce connected app. See xref:salesforce-connector-config-topics.adoc#create-consumer-key[Create a Consumer Key].
+* *Key store* (required): See xref:salesforce-connector-config-topics.adoc#keystoregen[Generate a Keystore File].
+* *Store password* (required): The password for the keystore.
+* *Principal* (required) The password for the keystore.
+
+== OAuth Username Password
+
+Important fields:
+
+* *Consumer key* (required): The consumer key for the Salesforce connected app. See xref:salesforce-connector-config-topics.adoc#create-consumer-key[Create a Consumer Key].
+* *Consumer secret* (required): The consumer secret for the connector to access Salesforce.
+* *Username* (required): Enter the Salesforce username.
+* *Password* (required): Enter the corresponding password.
+* *Security token*: Enter the corresponding security token.
+
+[[propvals]]
+== Connector Property Values
```

Comment:
If these are examples, put example in the heading. How many more operations are there, in other words?